packages/react-graphql/package.json

```diff
@@ -25,7 +25,6 @@
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/react-graphql/README.md",
   "dependencies": {
     "@apollo/react-common": ">=3.0.0 <4.0.0",
-    "@apollo/react-components": ">=3.0.0 <4.0.0",
     "@apollo/react-hooks": ">=3.0.0 <4.0.0",
```

Comment:
Should this have been removed?

Reply:
Yes. Before this package was just re-exporting the `Query` component from Apollo which was imported from `@apollo/react-components`. Now that we're defining our own component the import can be safely removed.